reference/installation/_api.html

```diff
@@ -568,6 +568,22 @@ <h3 id="operator.tigera.io/v1.CalicoNetworkSpec">CalicoNetworkSpec
 <tbody>
 <tr>
 <td>
+<code>linuxDataplane</code><br>
+<em>
+<a href="#operator.tigera.io/v1.LinuxDataplaneOption">
+LinuxDataplaneOption
+</a>
+</em>
+</td>
+<td>
+<em>(Optional)</em>
+<p>LinuxDataplane is used to select the dataplane used for Linux nodes. In particular, it
+causes the operator to add required mounts and environment variables for the particular dataplane.
+If not specified, iptables mode is used.</p>
```

Comment:
Looking at https://deploy-preview-4494--calico-master.netlify.app/reference/installation/api#operator.tigera.io/v1.LinuxDataplaneOption
it would be nice if this description had "One of: Iptables, BPF" or something on the end

Reply:
Or maybe its not _this_ description - whereever the other fields are getting it from.

Reply:
Agreed; this is auto-generated.  Will need to update the help text in the operator repo.